plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/statistics/MetaDirStatisticsAccess.java

```diff
@@ -89,17 +90,19 @@ public void updateExtendedStatistics(
             ExtendedStatistics statistics)
     {
         Path metaPath = new Path(tableLocation, STATISTICS_META_DIR);
-        ensureDirectoryExists(session, metaPath);
         try {
             Path statisticsPath = new Path(metaPath, STATISTICS_FILE);
 
-            FileSystem fileSystem = hdfsEnvironment.getFileSystem(new HdfsContext(session), metaPath);
-            try (OutputStream outputStream = fileSystem.create(statisticsPath, true)) {
+            TrinoFileSystem fileSystem = fileSystemFactory.create(session);
+            try (OutputStream outputStream = fileSystem.newOutputFile(statisticsPath.toString()).createOrOverwrite(newSimpleAggregatedMemoryContext())) {
```

Comment:
After https://github.com/trinodb/trino/commit/1e7ded9b25d5646a91d58bd3e57997c8c4873aa6 there is a no-arg version of `createOrOverwrite()` and `create()`

Reply:
Removing this should fix the build error due to missing the `trino-memory-context` (won't be needed)